### PR TITLE
Docs Update from open-telemetry/opentelemetry-collector

### PR DESCRIPTION
**Pull request by austinlparker**
This is an automated pull request.

### CODE REVIEW

**Comment by chalin** [comment]
Pinging @mtwo @flands -- Shouldn't these auto-generated PRs just require a [RSLGTM](https://chromium.googlesource.com/chromiumos/docs/+/master/glossary.md) and a click of the "Squash and Merge" button? If not, what's the process, and how can I help get these into the repo more quickly?

/cc @austinlparker @tedsuo

File: content/en/docs/collector/custom-auth.md [hunk 1]
```diff
@@ -1,117 +0,0 @@
----
-title: "Building a custom authenticator"
```
Comment (chalin) [inline content/en/docs/collector/custom-auth.md:2]: Generally speaking, when pages disappear, we should add an appropriate redirect rule. (Related: #635)

So, in this case, if a reader was interested in the content of this page (and the _Authentication_ section on the parent page), where should they go to next for pertinent info?

/cc @bogdandrutu @tigrannajaryan

**Review by chalin — APPROVED** [review]

File: content/en/docs/collector/custom-auth.md (hunk 1 above)
Comment (chalin) [reply content/en/docs/collector/custom-auth.md:2]: > Generally speaking, when pages disappear, we should add an appropriate redirect rule. (Related: #635)

I've opened #638 to track this.

Comment (tigrannajaryan) [reply content/en/docs/collector/custom-auth.md:2]: Thanks. I think https://github.com/open-telemetry/opentelemetry.io/issues/668 is the right link.

Comment (chalin) [reply content/en/docs/collector/custom-auth.md:2]: Right, thanks!

**Pull request merged**